test/unit/ngsiv2/examples/contextRequests/updateContextProcessTimestamp1.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Is this file related with the PR? I mean, it mentions "process timestamp" plugin (that I think it was done in a previous PR). Moreover, the string updateContextExpressionPlugin1.json seems not being used in any test file in this PR...

Reply:
Sorry for that! This file came somehow from previous PR.

Reply:
Fixed in 0b20a6e218960a8a2732a3dbf78d7c4ac76b52a5

Reply:
The werid thing is that if tha file was missing in the previous PR, that PR should have failed due to lack in this file... Could have a look to explain this little mistery, pls? :)

Reply:
In the previous PR #2, originally I included for the unit tests a file called updateContextProcessTimestamp1.json. You suggested to use instead updateContextProcessTimestamp.json and I did the change.

The problem is that I had already started the work for this new PR #3 in a separate branch task/expressionPluginNgsi2 before you started providing your feedback to speed-up the following steps- Once you gave me the LGTM to PR#2, I merged the changes into in the new branch task/expressionPluginNgsi2 but the old updateContextProcessTimestamp1.json was not removed.

As you can see, the file was not missing in the previous PR because we changed its name. ;)

Reply:
Ok, mistery clarified :)

NTC